packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -245,7 +242,7 @@ public void runGuarded() {
               }
               while (!copiedOperationsQueue.isEmpty()) {
                 NativeUpdateOperation op = copiedOperationsQueue.remove();
-                ReactShadowNode shadowNode = mUIImplementation.resolveShadowNode(op.mViewTag);
+                var shadowNode = mUIImplementation.resolveShadowNode(op.mViewTag);
```

Comment:
What's the reason for this change? We don't really use `var` in our codebase (should we?).